app/src/pages/playground/InvocationParametersForm.tsx

```diff
@@ -0,0 +1,136 @@
+import React from "react";
+
+import { Flex, Slider, TextField } from "@arizeai/components";
+
+import { ModelConfig } from "@phoenix/store";
+import { Mutable } from "@phoenix/typeUtils";
+
+import { getInvocationParametersSchema } from "./playgroundUtils";
+import { InvocationParametersSchema } from "./schemas";
+
+/**
+ * Form field for a single invocation parameter.
+ *
+ * TODO(apowell): Should this be generic over the schema field data type? There
```

Comment:
What does this comment mean exactly? If it's a real todo might add an issue and link it here. The one below seems like a real follow up so might be nice to have the issue